python/setup.py

```diff
@@ -9,7 +9,7 @@
 openmmplumed_library_dir = '@OPENMMPLUMED_LIBRARY_DIR@'
 
 # setup extra compile and link arguments on Mac
-extra_compile_args = []
+extra_compile_args = ['-D_GLIBCXX_USE_CXX11_ABI=0']
```

Comment:
Could you remove this?  We're getting ready to build the next release of OpenMM and it's going to use the C++11 ABI, so this will break it.